src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssueViewModel.cs

```diff
@@ -0,0 +1,93 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.CompilerServices;
+using SonarLint.VisualStudio.Core.Analysis;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.Taint.TaintList.ViewModels
+{
+    internal interface ITaintIssueViewModel : INotifyPropertyChanged, IDisposable
+    {
+        IAnalysisIssueVisualization TaintIssueViz { get; }
```

Comment:
The volume of code was getting too large to manage conveniently for review purposes so I changed my approach. I'm adding the necessary classes for a minimal working tool window that will display something, then add functionality incrementally (additional columns, selection, filtering etc).

Breaking up the work should make it easier to review and to identify opportunities for refactoring. We'll also have a working end to end sooner.